app/controllers/SessionsController.scala

```diff
@@ -410,4 +420,14 @@ class SessionsController @Inject()(messagesApi: MessagesApi,
       .flashing("message" -> "Feedback form schedule initiated"))
   }
 
+  def shareContent(id: String): Action[AnyContent] = action.async { implicit request =>
+    if (id.length != 24) {

```

Comment:
What is != 24 used for?

Reply:
If the id of the BSON document entered in the URL is not equal to 24 then it throws an error that shows up on the browser. So if someone manually type it out it would show up an error in the browser. To avoid that we have checked and redirected the request accordingly.

Reply:
Is it really required? I mean we're already using getById(id) and it would say session not found if the id is wrong.